android/src/com/mapswithme/maps/MwmBroadcastReceiver.java

```diff
@@ -0,0 +1,22 @@
+package com.mapswithme.maps;
+
+import android.content.BroadcastReceiver;
+import android.content.Context;
+import android.content.Intent;
+
+public abstract class MwmBroadcastReceiver extends BroadcastReceiver {
+
+    protected abstract void onReceiveInitialized(Context context, Intent intent);
```

Comment:
Add @nonnull annotations to parameters please

Reply:
I've tried to do that but in this case there is a warning saying that parameters in super class are not annotated as `@NonNull`. As I remember at least `intent` can be `null`.

Reply:
You are right, intent might be null. But as long as onRecieveInitialized is our method we can explicitly make "nullness" contract by ourself. If we decide to pass intent as non-null only you can check it on null before calling the onReceiveInitialized method either we can mark it @Nullable, both options are looked good for me besides the leaving parameters without annotations. I would want to follow our practise which we introduced in mapsme: mark all reference type parameters and fields with nullness annotations (where it's not obvious what value can be come from android sdk or from somewhere - it's good reason to mark as @Nullable, i.e. be prepared to process null in runtime)

Reply:
As for context - it's should be @nonnull.

Reply:
Any news here? I removed a lot of code by #199. Please re-check.

Reply:
@alexzatsepin annotations have been added

Reply:
@rtsisyk I've rebased on master. Everything is fine.